tests/plugins/dc/test_dublincore.py

```diff
@@ -0,0 +1,63 @@
+from lxml import etree
+import pytest
+from unittest import TestCase
+
+import metsrw
+import metsrw.plugins.dcrw as dcrw
+import tests.helpers as h
+
+
+class TestDublinCoreXmlData(TestCase):
```

Comment:
These tests live in `tests/plugins/dc`. Shouldn't it be `tests/plugins/dcrw`?

Reply:
yes, yes it should.